script.js

```diff
@@ -14,6 +14,294 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+    image: './img/SnapshootPortfolio4.png',
+    imgDesk: './img/SnapshootPortfolio4.png',
+    technologies: {
+      tech1: 'HTML',
+      tech2: 'CSS',
+      tech3: 'JavaScript',
+    },
+    counter: './img/Counter.png',
+    role: 'Back and dev',
+    company: 'canopy',
+    year: 2015,
+  },
+];
+
+const cardContainer = document.getElementById('card-container-one');
+
+const renderFirstCards = (firstCards, cardContainer) => {
+  firstCards.forEach((item) => {
+    const cardLayout = document.createElement('div');
+    cardLayout.id = `card${item.id}`;
+    cardLayout.innerHTML = ` 
+    <div class="works  works-two aling-self-center m-5 d-flex  justify-content-center">
+            <img src=${item.image} class="snapshoot w-75">
+            <img src=${item.imgDesk} class="photo-one">
+            <div class="left-block">
+                <div class="primary-text-block">
+                    <h2 class="project-title">${item.name}</h2>
+                    <h2 class="project-title-desktop">${item.nameDesk}</h2>
+                    <div class="frame-2">
+                        <h6 class="client">${item.company}</h6>
+                        <img src="img/Counter.png" class="counter">
+                        <p class= "client-role">${item.role}</p>
+                        <img src="img/Counter.png" class="counter">
+                        <p class="client-year">${item.year}</p>
+                    </div>
+                </div>
+                <p class= "primary-text mx-2 my-3">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+                <ul class="tags-block mt-3">
+                    <li class="list-item"><button type="button" class= "btn btn-tags">${item.technologies.tech1}</button></li>
+                    <li class="list-item"><button type="button" class= "btn btn-tags"> ${item.technologies.tech2}</button></li>
+                    <li class="list-item"><button type="button" class= " btn btn-tags">${item.technologies.tech3}</button></li>
+                </ul>
+                <div class="action">
+                    <button type="button" class="btn normal-b" data-modal-target="#popup${item.id}" > See project</button>
+                </div>
+            </div>
+        </div>
+    `;
+    cardContainer.appendChild(cardLayout);
+  });
+};
+
+renderFirstCards(firstCards, cardContainer);
+
+const projectInfo = [
+  {
+    id: 0,
+    name: 'Tonic',
+    nameDesk: 'Tonic',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry`s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: './img/SnapshootPortfolio.png',
+    technologies: {
+      tech1: 'HTML',
+      tech2: 'CSS',
+      tech3: 'JavaScript',
```

Comment:
Great job team 🤝

- [ ] As stated by the previous reviewer, kindly increase the height of your popup image so that your design doesn't appear distorted with a blurred image.
![image-desktop](https://user-images.githubusercontent.com/63917725/224520618-eab10230-2868-4cbc-b43c-f33143433463.PNG)

- [ ] [Optional] Kindly fix the responsiveness of your design so that your popup can adapt to different screen sizes.
![responsiveness](https://user-images.githubusercontent.com/63917725/224520653-152bbd51-5644-44a7-adea-1a6d18c32963.PNG)